module/PowerShellEditorServices/PowerShellEditorServices.psm1

```diff
@@ -37,9 +37,15 @@ function Start-EditorServicesHost {
         [string]
         $LanguageServiceNamedPipe,
 
+        [string]
+        $LanguageServiceWriteNamedPipe,
+
         [string]
         $DebugServiceNamedPipe,
 
+        [string]
+        $DebugServiceWriteNamedPipe,
```

Comment:
I think this might be fine for this PR, but I just wanted to call out that this param block (and the one in `Start-EditorServices.ps1`) should be using ParameterSets to better organize all these parameters.

Reply:
Yup, `-Stdio` should be in one parameter set, `-LanguageServiceNamedPipe/DebugServiceNamedPipe` in another and `-LanguageServiceWriteNamedPipe/DebugServiceWriteNamedPipe` in yet another where it seems like there should also be -LanguageServiceReadNamedPipe/DebugServiceReadNamedPipe`.  That would make the parameters a bit clearer IMO.

Reply:
Nice suggestions, about parameter sets, indeed, I'll correct this.